client/src/state/history/readOnePageQuery.js

```diff
@@ -0,0 +1,97 @@
+import { graphql } from 'react-apollo';
+import gql from 'graphql-tag';
+
+// GraphQL query for retrieving the version history of a specific page. The
+// results of the query must be set to the "versions" prop on the component
+// that this HOC is applied to (see PageHistoryViewer.js) for binding
+// implementation.
+const query = gql`
+query readOnePage ($page_id: ID!, $limit: Int!, $offset: Int!) {
```

Comment:
Minor point here, but these query names are arbitrary. They don't necessarily have to be consistent with their root query field name. For debugging purposes, it would probably help to give this a more distinct name like `query ReadHistoryViewerPage ($page_id: ID! ...)`

Reply:
Cool, will update